pkg/database/realm_stats.go

```diff
@@ -21,17 +21,24 @@ import (
 	"fmt"
 	"sort"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/google/exposure-notifications-verification-server/internal/icsv"
 	"github.com/google/exposure-notifications-verification-server/internal/project"
+	"github.com/lib/pq"
 )
 
 var _ icsv.Marshaler = (RealmStats)(nil)
 
 // RealmStats represents a logical collection of stats of a realm.
 type RealmStats []*RealmStat
 
+var claimDistributionBuckets = []time.Duration{
+	time.Minute, 5 * time.Minute, 25 * time.Minute, 30 * time.Minute, time.Hour,
```

Comment:
Do we really expect people to care about 25 vs 30 min?

Reply:
In general from a stats perspective, these buckets are a little weird.

Reply:
that's supposed to be 15m. True log scale would be more statistically sound